### PR TITLE
Added Custom Element For Taxes and Fees

### PR DESCRIPTION
**Pull request by billy-hardy**

### CODE REVIEW

**Comment by Red-Thirteen** [comment]
this pr is shit

File: index.html [hunk 1]
```diff
@@ -29,9 +31,9 @@
 					</pre>
 				</td>
 			</tr>
-		</table>
-		Taxes: $<input id="taxes" type="text"><br>
-		Fees: $<input id="fees" type="text"><br>
+        </table>
+        <!-- gusbus -->
```
Comment (Red-Thirteen) [inline index.html]: amazing

**Comment by billy-hardy** [comment]
fixes #2

**Comment by GustavoFernandes** [comment]
Outdated and unnecessary right now

**Pull request closed without merging**